README.md

```diff
@@ -1,8 +1,274 @@
+- [What the process is]
+- [The RFC life-cycle]
+- [Reviewing RFCs]
+- [Implementing an RFC]
+- [RFC Postponement]
+- [Help this is all too informal!]
+- [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Amethyst, its component crates, or the RFC process itself. The definition of substantial
+is somewhat fluid, and to an extent set by the community. Some examples might include:
+
+- A change to a different crate used for physics calculations
+- Changes to how the editor communicates with the engine
+- Significant protocol changes to the networking library
+- Anything that has the potential to produce a substantial negative impact on users
+
+Some changes do not require an RFC:
+
+- Rephrasing, reorganizing, refactoring, or otherwise "changing shape does
+  not change meaning".
+- Additions that strictly improve objective, numerical quality criteria
+  (warning removal, speedup, better platform coverage, more parallelism, trap
+  more errors, etc.)
+- Additions only likely to be _noticed by_ other developers-of-amethyst,
+  invisible to users-of-amethyst.
+- More tests and benchmarks
+
+If you submit a pull request that ends up being generating a lot of discussion or
+pushback, you may be asked to submit an RFC. What seems to be a minor change can
+have unintended consequences other teams might see.
+
+### Sub-team specific guidelines
+[Sub-team specific guidelines]: #sub-team-specific-guidelines
+
+For more details on when an RFC is required for the following areas, please see 
+the Amethyst community's [sub-team] specific guidelines for:
+
+- [networking changes](net_changes.md),
+- [renderer changes](rend_changes.md),
+- [engine changes](eng_changes.md),
+- [web changes](web_changes.md).
+
+## Before creating an RFC
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality
+proposals, proposals for previously-rejected features, or those that don't fit
+into the near-term roadmap, may be quickly rejected, which can be demotivating
+for the unprepared contributor. Laying some groundwork ahead of the RFC can
+make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is
+generally a good idea to pursue feedback from other project developers
+beforehand, to ascertain that the RFC may be desirable; having a consistent
+impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking
+the idea over on the [Discord channel](https://discord.gg/dqe9UYE), specifically with the applicable sub-team.
+
+As a rule of thumb, receiving encouraging feedback from long-standing project
+developers, and particularly members of the relevant [sub-team] is a good
+indication that the RFC is worth pursuing.
```

Comment:
I do think we should also add here in the end that you should never be afraid of attempting to bring up changes no matter how stupid it can be. I feel like just reading this may feel very discouraging, but we're usually pretty open and try to make sure that everyone's opinion is being taken into account.

Reply:
I'll reword this part